src/app/shared/comcol-forms/delete-comcol-page/delete-comcol-page.component.ts

```diff
@@ -65,4 +74,23 @@ export class DeleteComColPageComponent<TDomain extends DSpaceObject> implements
   onCancel(dso: TDomain) {
     this.router.navigate([this.frontendURL + '/' + dso.uuid + '/edit']);
   }
+
+  private refreshCache(dso: TDomain) {
```

Comment:
These two methods seem identical to the ones in `create-comcol-page.component.ts`.   Just wondering if we should find a way to place these methods in one place so both components can use the same method?  It'd be easier to maintain if there was no duplication here.

Reply:
I agree. It looks like refreshCache and parentCommunityUrl can be moved to ComColDataService. Both components already have a reference to it in `this.dsoDataService`.

Reply:
Thank you, I moved this function to ComColDataService and amended tests accordingly.